usmqe_tests/api/tendrl/others/test_user.py

```diff
@@ -0,0 +1,200 @@
+# -*- coding: utf8 -*-
+"""
+REST API test suite - user
+"""
+
+import json
+import pytest
+from usmqe.api.tendrlapi import user as tendrlapi_user
+from usmqe.api.tendrlapi.common import login, logout
+
+
+LOGGER = pytest.get_logger('user_test', module=True)
+"""@pylatest default
+Setup
+=====
+
+Prepare USM cluster accordingly to documentation.
+
+Further mentioned ``APIURL`` points to: ``http://USMSERVER/api/1.0``.
+"""
+
+"""@pylatest default
+Teardown
+========
+"""
+
+
+def test_user_get(valid_session_credentials):
+    """@pylatest api/user.get
+    API-users: get user
+    *******************
+
+    .. test_metadata:: author mkudlej@redhat.com dahorak@redhat.com fbalak@redhat.com
+
+    Description
+    ===========
+
+    Get user admin.
+    """
+    test = tendrlapi_user.ApiUser(auth=valid_session_credentials)
+    """@pylatest api/user.get
+    .. test_step:: 2
+
+        Send **GET** request to ``APIURL/users``.
+
+    .. test_result:: 2
+
+        List of users in database is returned
+    """
+    test.users()
+    """@pylatest api/user.get
+    .. test_step:: 3
+
+        Get user info.
+
+        Send **GET** request to ``APIURL/users/admin``.
+
+    .. test_result:: 3
+
+        User information for user *admin* is returned.
+    """
+    test.user('admin')
+
+
+def test_user_change_password(valid_new_user, valid_password):
```

Comment:
I really don't like to have a fixture for everything. E.g. valid password could be parametrised for the test itself as it is only this test which uses it. No need for another parametrised fixture, but as it seems I am in minority with this attitude.
Actually there's a possibility to use it elsewhere, but not as it is. See my comment for usmqe_tests/api/tendrl/others/conftest.py